app/src/main/res/menu/nav_feed_context.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
-
+    <item
+        android:id="@+id/multi_select"
+        android:menuCategory="container"
+        android:title="@string/multi_select" />
```

Comment:
Is this also shown in the side bar?

I would move it to the bottom for now.

Reply:
Got it.